tests/python_test.py

```diff
@@ -260,14 +258,9 @@ def test_read_resource_owners_invalid_align_size(tmpdir, sanlock_daemon):
     assert e.value.errno == errno.EINVAL
 
 
-@pytest.mark.parametrize("size,offset", [
-    # Smallest offset.
-    (MIN_RES_SIZE, 0),
-    # Large offset.
-    (LARGE_FILE_SIZE, LARGE_FILE_SIZE - MIN_RES_SIZE),
-])
-def test_add_rem_lockspace(tmpdir, sanlock_daemon, size, offset):
-    path = str(tmpdir.join("ls_name"))
+@pytest.mark.parametrize("size,offset,filename,encoding", TestParams) 
```

Comment:
This is slow test, and we don't need to add more permutations here.

If you want to test additional apis with unicode file names add separate fast tests for every api.

It will be nice also to remote the size parameters from this test and test them separately with
other calls, but it is not related to python 3 so lets not spend time on it now.